### ISSUE TITLE
throw er; // Unhandled 'error' event (LOCK: already held by process)

### ISSUE THREAD

**Issue by yuilleb** [Storj/core#502]
### Package Versions

```
StorjShare: 7.0.1
Core:       4.0.2
Protocol:   0.9.0
```

Replace the values below using the output from `node --version`.

```
v6.6.0
```

### Actual Behavior

```
{"level":"info","message":"received valid message from {\"userAgent\":\"4.0.2\",\"protocol\":\"0.9.0\",\"address\":\"212.24.106.202\",\"port\":4000,\"nodeID\":\"5b5a9ab1a584477fe0c775bf4dc3c8d1f2820fbf\",\"lastSeen\":1476347988469}","timestamp":"2016-10-13T08:39:48.469Z"}
{"level":"debug","message":"updating contact {\"userAgent\":\"4.0.2\",\"protocol\":\"0.9.0\",\"address\":\"212.24.106.202\",\"port\":4000,\"nodeID\":\"5b5a9ab1a584477fe0c775bf4dc3c8d1f2820fbf\",\"lastSeen\":1476347988469}","timestamp":"2016-10-13T08:39:48.470Z"}
{"level":"debug","message":"contact already in bucket, moving to tail","timestamp":"2016-10-13T08:39:48.470Z"}
{"level":"debug","message":"checking pending rpc callback stack for abd6fdc231ea963d2f401c425cb9cec2361320fd","timestamp":"2016-10-13T08:39:48.496Z"}
{"level":"info","message":"replying to message to 1862c6344cf3b640f7fa5eda1074b07134bc8324","timestamp":"2016-10-13T08:39:48.496Z"}
{"level":"debug","message":"not waiting on callback for message 1862c6344cf3b640f7fa5eda1074b07134bc8324","timestamp":"2016-10-13T08:39:48.506Z"}
{"level":"warn","message":"error in storage manager: IO error: lock /mnt/md3/storj/sharddata.kfs/112.s/LOCK: already held by process","timestamp":"2016-10-13T08:39:49.203Z"}
{"level":"warn","message":"error in storage manager: IO error: lock /mnt/md3/storj/sharddata.kfs/112.s/LOCK: already held by process","timestamp":"2016-10-13T08:39:49.203Z"}
{"level":"warn","message":"error in storage manager: IO error: lock /mnt/md3/storj/sharddata.kfs/112.s/LOCK: already held by process","timestamp":"2016-10-13T08:39:49.203Z"}
{"level":"warn","message":"error in storage manager: IO error: lock /mnt/md3/storj/sharddata.kfs/112.s/LOCK: already held by process","timestamp":"2016-10-13T08:39:49.204Z"}
{"level":"warn","message":"error in storage manager: IO error: lock /mnt/md3/storj/sharddata.kfs/112.s/LOCK: already held by process","timestamp":"2016-10-13T08:39:49.204Z"}
events.js:160
      throw er; // Unhandled 'error' event
      ^

Error: IO error: lock /mnt/md3/storj/sharddata.kfs/112.s/LOCK: already held by process
    at Error (native)
```

**Comment by bookchin** [comment]
Duplicate of https://github.com/Storj/storjshare-cli/issues/191
